### PR TITLE
CSS maintenance: resolved value

### PR DESCRIPTION
**Pull request by estelle**
expanded on the definition.
edited the see also section to be more relevant to someone reading the current page.

### CODE REVIEW

**Comment by Josh-Cena** [comment]
While you are on it, not sure if you are aware of this issue: https://github.com/mdn/content/issues/21142 maybe it's a good time to restructure?

File: files/en-us/web/css/resolved_value/index.md [hunk 1]
```diff
@@ -7,32 +7,24 @@ spec-urls: https://drafts.csswg.org/cssom/#resolved-values
...
+The **resolved value** of a [CSS](/en-US/docs/Web/CSS) property is the value returned by the {{domxref("Window.getComputedStyle", "getComputedStyle()")}} method, which returns a live {{domxref("CSSStyleDeclaration")}} object containing the values of all CSS properties of an element, after applying active stylesheets and resolving any basic computation those values may contain.
 
-For most properties, it is the [computed value](/en-US/docs/Web/CSS/computed_value), but for a few legacy properties (including {{cssxref("width")}} and {{cssxref("height")}}), it is instead the [used value](/en-US/docs/Web/CSS/used_value). See the specification link below for more per-property details.
+Historically, `getComputedStyle()` was defined to return the [computed value](/en-US/docs/Web/CSS/computed_value) of an element or pseudo-element. As CSS evolved, so did the concept of "computed value", but the implementation of `getComputedStyle()` had to remain the same for backward compatibility with deployed scripts. The "resolved value" resolves this.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/resolved_value/index.md:12]:
```suggestion
Historically, `getComputedStyle()` returned the [computed value](/en-US/docs/Web/CSS/computed_value) of an element or pseudo-element. As CSS evolved, so did the concept of "computed value", but the values returned by `getComputedStyle()` had to remain the same for backward compatibility with deployed scripts. These values are the "resolved values".
```

File: files/en-us/web/css/resolved_value/index.md [hunk 2]
```diff
@@ -7,32 +7,24 @@ spec-urls: https://drafts.csswg.org/cssom/#resolved-values
...
+
+For most properties, it is the [computed value](/en-US/docs/Web/CSS/computed_value), but for a few legacy properties (including {{cssxref("width")}} and {{cssxref("height")}}), it is instead the [used value](/en-US/docs/Web/CSS/used_value). See the specification link below for some per-property details.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/resolved_value/index.md:14]:
```suggestion
For most properties, the resolved value is the [computed value](/en-US/docs/Web/CSS/computed_value), but for a few legacy properties (including {{cssxref("width")}} and {{cssxref("height")}}), it is the [used value](/en-US/docs/Web/CSS/used_value). See the specification link below for some per-property details.
```

**Review by chrisdavidmills — APPROVED** [review]
Mostly looking good @estelle. A few bits for your consideration; approving.

**Comment by estelle** [comment]
> While you are on it, not sure if you are aware of this issue: #21142 maybe it's a good time to restructure?

I plan on doing css values very soon, so will do it then. thanks!

**Pull request merged**